apps/api/src/routers/admin.py

```diff
@@ -172,6 +172,47 @@ async def submit_review(
         raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+@router.post("/review/hacker")
+async def submit_hacker_review(
+    applicant: str = Body(),
+    score: float = Body(),
+    reviewer: User = Depends(require_role({Role.REVIEWER})),
+) -> None:
+    """Submit a review decision from the reviewer for the given hacker applicant."""
+    log.info("%s reviewed hacker %s", reviewer, applicant)
+
+    review: HackerReview = (utc_now(), reviewer.uid, score)
+
+    try:
+        await mongodb_handler.raw_update_one(
+            Collection.USERS,
+            {"_id": applicant},
+            {"$push": {"application_data.reviews": review}},
+        )
+    except RuntimeError:
+        log.error("Could not submit review for %s", applicant)
+        raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    applicant_record = await mongodb_handler.retrieve_one(
+        Collection.USERS,
+        {"_id": applicant},
+        ["_id", "application_data.reviews"],
+    )
+    if not applicant_record:
+        log.error("Could not retrieve applicant after submitting review")
+        raise HTTPException(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    num_reviewers = applicant_review_processor.get_num_unique_reviewers(
+        applicant_record
+    )
+    if num_reviewers > 1:
```

Comment:
Suggestion: could you include a comment above this condition explaining that the status is set when there are at least two reviewers and/or use `>= 2` instead of `> 1`? Took me a moment to understand why this was moved out.